apps/www/pages/politikfragebogen/overview.js

```diff
@@ -0,0 +1,63 @@
+import { createGetServerSideProps } from '../../lib/apollo/helpers'
+import SubmissionsOverview from '../../components/PoliticsQuestionnaire/Overview'
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { csvParse } from 'd3-dsv'
+import { nest } from 'd3-collection'
+
+import { QUESTION_TYPES } from '../../../www/components/PoliticsQuestionnaire/config'
+
+import { leftJoin } from '../../../www/components/PoliticsQuestionnaire/utils'
+
+export default ({ submissionData }) => (
+  <SubmissionsOverview submissionData={submissionData} />
+)
+
+export const getServerSideProps = createGetServerSideProps(
+  async ({
+    ctx: {
+      query: { party },
+    },
+  }) => {
+    const data = await fs.readFile(
+      path.join(
+        process.cwd(),
+        'public/static/politicsquestionnaire2023/submissions_data.csv',
```

Comment:
Rather than loading the path via the CWD, perhaps it would make more sense,
to load it via PUBLIC_BASE_URL?